src/components/Media/InfoPanel.js

```diff
@@ -0,0 +1,62 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+import { MdCheck, MdClose, MdDelete, MdPlayArrow, MdStar } from "react-icons/md";
+
+const useStyles = makeStyles((theme) => ({
+  line: {
+    display: "flex",
+  },
+  icon: {
+    fontSize: "1.5rem",
+  },
+  text: {
+    width: "95%",
+    paddingLeft: theme.spacing(3),
+    paddingBottom: theme.spacing(3),
+  },
+  divider: {
+    width: "2px",
+  },
+}));
+
+function InfoPanel() {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <div className={classes.line}>
+        <MdCheck className={classes.icon} />
```

Comment:
Die Icons sind ja oben bündig und hängen deswegen so in der Luft. Vlt. könnte man die Icons mittig zum Text zentrieren, dann sähe es einheitlicher aus.

Reply:
Wir haben zusammen entschieden, es so zu lassen